### PR TITLE
header links are added for texts and image, header z-index is reduced to enable sidekick to appear above the header, header image size updated

### LINKED ISSUE #65: Menus in the header are not functional

### PR DESCRIPTION
**Pull request by kalimuthu-a**
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fix #65

Test URLs:
- Before: https://header-link-fixes--airindia--hlxsites.hlx.live/in/en/book/special-offers/global-connectivity
- After: https://main--airindia--hlxsites.hlx.live/in/en/book/special-offers/global-connectivity

### CODE REVIEW

File: blocks/header/header.css [hunk 1]
```diff
@@ -1015,3 +1054,79 @@ header .block .nav-wrapper.expanded nav .nav-sections .nav-drop .link-title {
...
+  91.67% {
+    background: #252f3a;
+    border-radius: 2px;
+    color: #fff;
+    transform: rotateY(540deg)
+  }
+
+  100.0% {
+    background: linear-gradient(100deg, #b61132 -9.01%, #82102c 92.66%);
+    color: #fff;
```
Comment (sselvara) [inline blocks/header/header.css:1099]: use color variables pls

File: blocks/header/header.css [hunk 2]
```diff
@@ -1015,3 +1054,79 @@ header .block .nav-wrapper.expanded nav .nav-sections .nav-drop .link-title {
...
+    background: -webkit-gradient(linear, left top, right top, color-stop(-321.46%, #fed9a0), color-stop(99.9%, #ebbe69));
+    background: linear-gradient(90deg, #fed9a0 -321.46%, #ebbe69 99.9%);
+    color: #1c1213;
+    transform: rotateY(1turn)
+  }
+
+  75.0%,
+  83.33%,
+  91.67% {
+    background: #252f3a;
```
Comment (sselvara) [inline blocks/header/header.css:1091]: use color variables pls

File: blocks/header/header.css [hunk 3]
```diff
@@ -1015,3 +1054,79 @@ header .block .nav-wrapper.expanded nav .nav-sections .nav-drop .link-title {
...
+  8.33% {
+    background: linear-gradient(100deg, #b61132 -9.01%, #82102c 92.66%);
+    color: #fff;
+    transform: rotateY(0deg)
+  }
+
+  25.0%,
+  33.33%,
+  41.67% {
+    background: linear-gradient(103deg, #eceef0 -131.29%, #bcc0be 109.54%);
```
Comment (sselvara) [inline blocks/header/header.css:1074]: use color variables pls

File: blocks/header/header.css [hunk 4]
```diff
@@ -1015,3 +1054,79 @@ header .block .nav-wrapper.expanded nav .nav-sections .nav-drop .link-title {
...
+
+/*
+ANIMATION
+**/
+
+@keyframes nav-badge {
+  0.0%,
+  16.67%,
+  8.33% {
+    background: linear-gradient(100deg, #b61132 -9.01%, #82102c 92.66%);
```
Comment (sselvara) [inline blocks/header/header.css:1066]: use color variables pls

File: blocks/header/header.css [hunk 5]
```diff
@@ -568,6 +568,41 @@ header .nav-wrapper .section.nav-tools .default-content-wrapper p a {
...
 
+.nav-badge {
+  -webkit-box-pack: center;
+  -ms-flex-pack: center;
+  -webkit-box-align: center;
+  -ms-flex-align: center;
+  align-items: center;
+  animation: nav-badge 12s linear infinite;
+  animation-fill-mode: forward;
+  background: linear-gradient(100deg, #b61132 -9.01%, #82102c 92.66%);
```
Comment (sselvara) [inline blocks/header/header.css:579]: use color variables pls

File: blocks/header/header.css [hunk 6]
```diff
@@ -568,6 +568,41 @@ header .nav-wrapper .section.nav-tools .default-content-wrapper p a {
   display: none;
 }
 
+.nav-badge {
+  -webkit-box-pack: center;
```
Comment (sselvara) [inline blocks/header/header.css:572]: do we need these vendor prefix?

**Review by sselvara — CHANGES_REQUESTED** [review]

File: blocks/header/header.css (hunk 1 above)
Comment (kalimuthu-a) [reply blocks/header/header.css]: fixed

File: blocks/header/header.css (hunk 2 above)
Comment (kalimuthu-a) [reply blocks/header/header.css]: fixed

File: blocks/header/header.css (hunk 3 above)
Comment (kalimuthu-a) [reply blocks/header/header.css]: fixed

File: blocks/header/header.css (hunk 4 above)
Comment (kalimuthu-a) [reply blocks/header/header.css]: fixed

File: blocks/header/header.css (hunk 5 above)
Comment (kalimuthu-a) [reply blocks/header/header.css]: fixed

File: blocks/header/header.css (hunk 6 above)
Comment (kalimuthu-a) [reply blocks/header/header.css]: fixed

**Review by sselvara — APPROVED** [review]

**Pull request merged**